File: 110algoritmos/exercicios017/ex017-Csharp/ex017.cs
using System;

class Program {
    static void Main(){
        double ca, co, hypo;

        Console.WriteLine("Digite o cateto adjacente: ");
        ca = double.Parse(Console.ReadLine());

        Console.WriteLine("Digite o cateto oposto: ");
        co = double.Parse(Console.ReadLine());

        hypo = Math.Sqrt(Math.Pow(ca, 2.0) + Math.Pow(co, 2.0));

        Console.WriteLine("A hypotenusa é: " + hypo);
    }
}

File: 110algoritmos/exercicios017/ex017-Rust/ex017.rs
use std::io;

fn main(){
    let mut ca:String = String::from("");
    let mut co:String = String::from("");

    println!("Digite o cateto adjacente: ");
    io::stdin()
        .read_line(&mut ca)
        .expect("Erro na leitura ca");
    let ca:f64 = ca.trim().parse().expect("Erro conversao ca");

    println!("Digite o cateto oposto: ");
    io::stdin()
        .read_line(&mut co)
        .expect("Error leitura co");
    let co:f64 = co.trim().parse().expect("Error conversao co");

    let hipo = ca.hypot(co);

    println!("A hipotenusa é igual a: {:.2}", hipo);

}